doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -435,64 +435,40 @@ Bluetooth Mesh samples
 
 * :ref:`ble_mesh_dfu_distributor` sample:
 
-  * Added:
-
-    * Support for pairing with display capability and the :ref:`bt_mesh_le_pair_resp_readme`.
-
-  * Fixed:
-
-    * An issue where the shell interface was not accessible over UART because UART was used as a transport for MCUmgr SMP protocol.
-      Shell is now accessible over RTT.
+  * Added support for pairing with display capability and the :ref:`bt_mesh_le_pair_resp_readme`.
+  * Fixed an issue where the shell interface was not accessible over UART because UART was used as a transport for MCUmgr SMP protocol.
```

Comment:
```suggestion
  * Fixed an issue where the shell interface was not accessible over UART because UART was used as a transport for the MCUmgr SMP protocol.
```